huaweicloud/services/dis/resource_huaweicloud_dis_stream.go

```diff
@@ -174,9 +176,9 @@ func ResourceDisStream() *schema.Resource {
 }
 
 func resourceDisStreamCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
func resourceStreamCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {

Reply:
It is already in the dis package, so there is no need to add dis to the function name, Please check all relevant areas.